File: Player.cpp
// Honor Pledge: I pledge that I have neither given nor received any help on this assignment

#include <iostream>
#include "Player.h"

//constructor
Player::Player() {

}

//deconstructor
Player::~Player()
{

}

//setters and getters to retrieve private nm
void Player::setNum(int num) {
  num = number;
}

int Player::getNum() {
  return number;
}

//function to determine if Player 1 hit or missed
int Player::shootBall1(int randNum) {
    if (randNum == 1){
      std::cout<<"Hit shot!"<<std::endl;
    } else {
      std::cout<<"Missed shot!"<<std::endl;
    }

}
//function to determine if Player 2 hit or missed
int Player::shootBall2(int randNum2) {
  if (randNum2 == 1){
    std::cout<<"Hit shot!"<<std::endl;
  } else {
    std::cout<<"Missed shot!"<<std::endl;
  }

}

File: Player.java
// Honor Pledge: I pledge that I have neither given nor received any help on this assignment

import java.io.*;

public class Player {

      //constructor
      public Player() {}

      //used to store the rand number
      private int num;

      //setters and getters to get private number
      public Player(int num) {
        this.num = num;
      }

      public int getNum() {
        return this.num;
      }

      //function to simulate shooting the ball for each player
      //will return if the player hit or missed the shot
      public void shootBall1(int randNum){
        if (randNum == 1) {
          System.out.println("Hit shot!");
        } else {
          System.out.println("Missed shot!");
        }
      }

      public void shootBall2(int randNum2){
        if (randNum2 == 1) {
          System.out.println("Hit shot!");
        } else {
          System.out.println("Missed shot!");
        }
    }
}
